NEWS.md

```diff
@@ -1,9 +1,41 @@
-# ospsuite.reportingengine 2.0.189
+# ospsuite.reportingengine 2.1.0
+
+## New features
+
+- Population workflows can use Monte Carlo sampling to handle ratio comparison (#536) as documented in article [PK Parameters in Population Workflows](../articles/pop-pk-parameters.html#ratio-comparison).

```

Comment:
`as documented in article [PK Parameters in Population Workflows]`
should be "in **the** article" I think